i18n-helpers/src/gettext.rs

```diff
@@ -38,11 +40,21 @@ fn strip_formatting(text: &str) -> String {
         .collect()
 }
 
-fn translate(text: &str, catalog: &Catalog) -> String {
+fn translate(text: &str, catalog: &Catalog) -> anyhow::Result<String> {
     let events = extract_events(text, None);
-    let translated_events = translate_events(&events, catalog);
-    let (translated, _) = reconstruct_markdown(&translated_events, None);
-    translated
+    // Translation should always succeed.
+    let translated_events =
+        translate_events(&events, catalog).expect("Fatal: failed to translate events");
```

Comment:
Is this `Fatal:` prefix needed here? It seems to be inconsistent with the rest of `expect()` failures we have in this codebase.

Reply:
Right I can't see it anywhere else, removed it. And panic is already fatal anyway...